src/go/k8s/README.md

```diff
@@ -34,12 +35,35 @@ export KUBECONFIG=your/path/to/kubeconfig.yaml
 kind create cluster --config kind.yaml
 ```
 
-You can simply deploy the Redpanda operator by running the following command
+In order to have validating webhook the cert manager needs to be
+installed. You can do this by running the following command:
+
+```
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.2.0/cert-manager.yaml
+```
+
+The cert manager needs around 1 minute to be ready. The Redpanda
+operator will create Issuer and Certificate custom resource. The
+webhook of cert-manager will prevent from creating mentioned
+resources. To verify that cert manager is ready for handling
+new request please run the following command:
+
+```
+kubectl wait pod -l app.kubernetes.io/instance=cert-manager -n cert-manager --timeout=10m --for condition=ready
+```
+
+You can simply deploy the Redpanda operator with webhook (recommended) by running the following command
 
 ```
 kubectl apply -k https://github.com/vectorizedio/redpanda/src/go/k8s/config/default
 ```
 
+You can deploy the Redpanda operator without webhook by running the following command:
```

Comment:
I think this is absolutely not recommended, maybe only for development, but then we should not put it into readme and rather hide it. Just wondering why you think we should highlight this?

Reply:
Many projects doesn't require cert-manager as a dependency. I was looking how others describe installation process especially with cert-manager dependency. Maybe they don't use `webhooks` and validate each request in the reconciliation loop. The errors are visible in logs/events/CR status. 

Currently we only validate the cluster resource in webhooks. I see this as nice to have feature.

Reply:
do you have any examples? I feel like every bigger and serious project has webhook and recommends it.

cert-manager does it, rook does it https://github.com/rook/rook/blob/8d2e873f41fd25bd8d9855fc8691e54de152aebf/Documentation/admission-controller-usage.md I can continue...

Yeah right now the logic is pretty weak there but I would rather help people help deploy cert-manager as it's super-easy than help them run without it. I can be convinced but that's my opinion here... :)

Reply:
I agree that operator without webhook is a bad idea. I would like to move to Kubernetes way of creating certificates [as in your rook example](https://github.com/rook/rook/blob/8d2e873f41fd25bd8d9855fc8691e54de152aebf/tests/scripts/webhook-create-signed-cert.sh#L87-L100).

I can even think of decoupling webhook service from the operator itself if that could help with the adoption.

So my list of operators that doesn't require cert-manager:

- Aqua security https://github.com/aquasecurity/aqua-operator/blob/5.3.0/docs/Installation.md
- ArangoDB https://github.com/arangodb/kube-arangodb#installation-of-latest-release-using-kubectl
- ArgoCD https://argocd-operator.readthedocs.io/en/latest/install/manual/
- Casandra operator https://github.com/instaclustr/cassandra-operator/wiki/Installation-and-deployment
- CockroachDB Kubernetes Operator https://github.com/cockroachdb/cockroach-operator#install-the-operator
- Jaegertracing Operator https://github.com/jaegertracing/jaeger-operator#getting-started
- Crossplane https://crossplane.io/docs/v1.0/getting-started/install-configure.html
- Datadog operator https://docs.datadoghq.com/agent/kubernetes/?tab=operator
- Elastic cloud on Kubernetes https://www.elastic.co/guide/en/cloud-on-k8s/1.4/k8s-deploy-eck.html
- Prometheus operator https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/user-guides/getting-started.md#example-prometheus-operator-manifest
- IBM cloud operator https://github.com/ibm/cloud-operators#prerequisites
- Istio operator https://istio.io/latest/docs/setup/install/operator/
- keycloak operator https://github.com/keycloak/keycloak-operator#deploying-to-a-cluster
- Knative operator https://github.com/knative/operator/blob/master/docs/installation.md 
- Kong operator https://docs.konghq.com/enterprise/2.3.x/deployment/installation/kong-for-kubernetes/ and https://github.com/Kong/kong-operator#quick-start
- Kubeflow https://www.kubeflow.org/docs/aws/deploy/install-kubeflow/
- minio operator https://github.com/minio/operator#prerequisites
- MongoDB operator https://github.com/mongodb/mongodb-kubernetes-operator/blob/master/docs/install-upgrade.md
- OpenEBS operator https://docs.openebs.io/docs/next/installation.html
- Postgresql operator https://github.com/zalando/postgres-operator/blob/master/docs/quickstart.md#deployment-options

The list that require cert-manager:

- Opentelemetry operator https://github.com/open-telemetry/opentelemetry-operator#getting-started

Reply:
Alright, I am convinced 👍  yeah let's remove the cert-manager dependency if that's blocker for folks - once we have people requesting it. Honestly, I think everyone has cert-manager at any serious cluster deployed

Reply:
I had experience with HashiCorp vault :) so there are alternatives cluster that doesn't have cert-manager.